test/integration/smart_answer_flows/estimate_self_assessment_penalties_test.rb

```diff
@@ -250,4 +250,138 @@ class EstimateSelfAssessmentPenaltiesTest < ActiveSupport::TestCase
       end
     end
   end
+
+  context "calculating late payment penalties" do
+    payment_amount = "100.00"
+
+    expected_penalty = {
+      step_1: 5,
+      step_2: 10,
+      step_3: 15,
+    }
+
+    penalty_dates = {
+      "2013-14": {
+        step_1: "2015-03-03",
+        step_2: "2015-08-03",
+        step_3: "2016-02-03",
+      },
+      "2014-15": {
+        step_1: "2016-03-02",
+        step_2: "2016-08-02",
+        step_3: "2017-02-02",
+      },
+      "2015-16": {
+        step_1: "2017-03-03",
+        step_2: "2017-08-03",
+        step_3: "2018-02-03",
+      },
+      "2016-17": {
+        step_1: "2018-03-03",
+        step_2: "2018-08-03",
+        step_3: "2019-02-03",
+      },
+      "2017-18": {
+        step_1: "2019-03-03",
+        step_2: "2019-08-03",
+        step_3: "2020-02-03",
+      },
+      "2018-19": {
+        step_1: "2020-03-02",
+        step_2: "2020-08-02",
+        step_3: "2021-02-02",
+      },
+      "2019-20": {
+        step_1: "2021-04-02",
```

Comment:
Is this April rather than March because of this?

> Penalty 1 = 1st late payment penalty (normally payment 30 days late, but not for 2019/20 due to extension of payment date due to Covid)

If so, maybe an inline comment explaining the difference for this one? Also not 100% sure where you got the date, as it appears to be missing from the Trello card.

Reply:
The dates were copied from the Zendesk ticket.  There's a table in the message dated 20 May 16:17.  I'll add a comment.